HT4_part1.py

```diff
@@ -0,0 +1,59 @@
+# import modules random and string
+import random
+import string
+
+def generate_random_dicts(dict_num_min,
+                          dict_num_max,
+                          key_num_min,
+                          key_num_max,
+                          value_min,
+                          value_max):
+    global list_of_dicts

```

Comment:
Keyword `global` usually uses in a different way. `global` allows to expand visibility area of already defined parameter (for example configuration parameter like database_name or endpoint to api, etc.)
Let's think about: What will change if we remove this line? Actually, nothing, because output of the function can be written to the variable in the part with main logic.
Let's think from other side for additional understanding: is the function changing some data/objects in outside the script? I believe no, because it just generate list of dicts based on rules you coded and function will always return random list of dicts in line with those rules. Rules are not rely on any object/data outside the script, so, it is not necessary to have any global variable/config constant.